src/routes/tools/tools.json

```diff
@@ -322,5 +322,15 @@
 		"addedOn": "2021-12-08",
 		"tags": [],
 		"stars": 0
-	}
+	},
+	{
+		"title": "walk-and-graph-svelte-components",
+		"url": "https://github.com/j2l/walk-and-graph-svelte-components",
+		"description": "CLI node script to walk svelte and js files, to draw a beautiful JPG of your dependencies aka \"imports\". No external dependency, only FS (Node). ",
+		"npm": "none",
```

Comment:
Instead of `"npm": "none",` you can completely remove this line

Reply:
Ok